VideoLocker/res/values/styles.xml

```diff
@@ -98,4 +98,8 @@
         <item name="android:textSize">12sp</item>
     </style>
 
+    <style name="AppTheme.Dark" parent="@android:style/Theme.Holo.Light">
+        <item name="android:actionBarItemBackground">@color/blue_bar</item>
```

Comment:
I should have brought this up before, but maybe we should rename "blue_bar" to be something more meaningful, like "title_color".